x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/ilm/IndexLifecycleService.java

```diff
@@ -208,9 +212,9 @@ void onMaster(ClusterState clusterState) {
                     } catch (Exception e) {
                         if (logger.isTraceEnabled()) {
                             logger.warn(
-                                () -> format(
+                                new ParameterizedMessage(
```

Comment:
Thise lambda to `ParameterizedMessage` look like a bad merge to me, please undo it.